jenkins/Jenkinsfile

```diff
@@ -0,0 +1,109 @@
+def secrets = [
+  [path: 'secret/jenkins/github', engineVersion: 2, secretValues: [
+    [envVar: 'PRIVATE_REPO_GITHUB_TOKEN', vaultKey: 'private_token']]],
+  [path: 'secret/jenkins/npm', engineVersion: 2, secretValues: [
+    [envVar: 'NPM_TOKEN', vaultKey: 'token']]],
+]
+
+def configuration = [vaultUrl: "${env.VAULT_URL}",  vaultCredentialId: 'vault-app-role', engineVersion: 2]
+
+pipeline {
+  agent {
+    kubernetes {
+      label 'common-ui'
+      defaultContainer 'jnlp'
+      yamlFile 'jenkins/build-pod.yaml'
+    }
+  }
+  
+  environment {
+    MAIN_BRANCH = 'master'
+    WORKSPACE = "${env.WORKSPACE}"
+    BRANCH = "${env.BRANCH}"
+    PROJECT = "cloudify-ui-common"
+  }
+
+  options {
+    checkoutToSubdirectory('cloudify-ui-common')
+    buildDiscarder(logRotator(numToKeepStr:'30'))
+    timeout(time: 60, unit: 'MINUTES')
+    timestamps()
+  }
+  stages {
+    stage('Build') {
+      steps {
+        container('node'){
+          dir("${env.WORKSPACE}/${env.PROJECT}") {
+            echo "Install dependencies"
+            sh 'npm ci --prefer-offline'
+            
+            echo "Build application"
+            sh 'npm run build'
+
+            echo "Run static analysis and check code style"
+            sh 'npm run lint'
+
+            echo "Run tests with coverage check"
+            sh 'npm run test:coverage'
+
+            echo "Run tests"
+            sh 'npm run test'
```

Comment:
It's not necessary to run `npm run test` here.

The scripts configuration right now are that:
```
        "test": "npm run lint && npm run test:coverage"
```

so both - `lint` and `test:coverage` - in `test` script are already part of the execution.

Reply:
ok, removing